src/profiler/profiler.c

```diff
@@ -422,7 +422,7 @@ void xdebug_profiler_function_end(function_stack_entry *fse)
 		fse->profile.time -= call_entry->time_taken;
 		fse->profile.memory -= call_entry->mem_used;
 	}
-	fprintf(XG_PROF(profile_file), "%d %lu %ld\n", fse->profiler.lineno, (unsigned long) (fse->profile.time * 1000000), (fse->profile.memory));
+	fprintf(XG_PROF(profile_file), "%d %lu %ld\n", fse->profiler.lineno, max(1, (unsigned long) (fse->profile.time * 1000000)), (fse->profile.memory));
```

Comment:
very important if consumed time was not possible to measure (because of low output time resolution (microsecond now) or low-resolution time source/measurement